core/templates/dev/head/pages/Base.ts

```diff
@@ -12,22 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+require('domain/sidebar/SidebarStatusService.ts');
+require('domain/utilities/UrlInterpolationService.ts');
+require('services/AlertsService.ts');
```

Comment:
Should this be sorted in the order commented [here](https://github.com/oppia/oppia/pull/6818/files#r289598937).
> vojtechjelinek: I meant even after UtilsService require. We first order by how deep is the file in the folder structure and then by alphabetical order.

Reply:
I think this is correct now, I didn't explain it well, it should be alphabetical order of the first folder → depth of the folder structure → alphabetical ordering of the other subfolders/filenames.